service/src/test/java/uk/gov/hmcts/reform/fpl/model/CaseManagementOrderTest.java

```diff
@@ -17,20 +17,20 @@
 
     @Test
     void shouldReturnTrueIfCaseManagementOrderStatusIsApproved() {
-        CaseManagementOrder caseManagementOrder = CaseManagementOrder.builder().status(APPROVED).build();
+        HearingOrder caseManagementOrder = HearingOrder.builder().status(APPROVED).build();
         assertThat(caseManagementOrder.isRemovable()).isTrue();
     }
 
     @ParameterizedTest
     @EnumSource(value = CMOStatus.class, names = {"SEND_TO_JUDGE", "RETURNED", "DRAFT"})
     void shouldReturnFalseIfCaseManagementOrderStatusIsNotApproved(CMOStatus cmoStatus) {
-        CaseManagementOrder caseManagementOrder = CaseManagementOrder.builder().status(cmoStatus).build();
+        HearingOrder caseManagementOrder = HearingOrder.builder().status(cmoStatus).build();
         assertThat(caseManagementOrder.isRemovable()).isFalse();
     }
 
     @Test
     void shouldFormatCaseManagementOrderAsLabel() {
-        CaseManagementOrder caseManagementOrder = CaseManagementOrder.builder().dateIssued(NOW).build();
+        HearingOrder caseManagementOrder = HearingOrder.builder().dateIssued(NOW).build();
 
         assertThat(caseManagementOrder.asLabel()).isEqualTo(
             String.format("Case management order - %s", formatLocalDateToString(NOW, "d MMMM yyyy")));
```

Comment:
vv minor but seeing as you have updated the file I will mention it. I would prefer a fixed date be used here and then we know exactly what the output would be an not have to rely on a formatting method. Up to you if you want to change it or not